cpanfile

```diff
@@ -44,6 +44,7 @@ requires 'XML::LibXML';
 requires 'XML::Simple';
 requires 'XML::Writer';
 requires 'YAML::PP';
+requires 'WWW::Mechanize';
```

Comment:
@okurz  can you confirm that this fine ?

Reply:
We're using now the `Mojo` dependency, which is already used.